sdks/python/apache_beam/coders/coders.py

```diff
@@ -1385,7 +1388,8 @@ def from_runner_api_parameter(payload, components, context):
         components[0],
         read_state=context.iterable_state_read,
         write_state=context.iterable_state_write,
-        write_state_threshold=int(payload))
+        write_state_threshold=int(payload)
```

Comment:
This isn't part of the documentation for the spec for the state backed iterable.